src/ua/dymohlo/reflection/PersonFactory.java

```diff
@@ -0,0 +1,34 @@
+package ua.dymohlo.reflection;
+/*
+Створіть клас PersonFactory, який має наступні методи:
+public Person createPerson(String name, int age, String address) - створює об'єкт типу Person із заданими параметрами та повертає його.
+public void printFields(Object obj) - використовуючи рефлексію, виводить на консоль всі поля та їх значення для об'єкта,
+який передається методу. Для визначення типу полів використовуйте метод getClass() або літерал класу MyClass.class.
+Створіть об'єкт класу PersonFactory і використайте його для створення об'єкту Person і виведення інформації про його поля.
+(створювати обєкти необхідно за допомогою рефлексії -- https://jenkov.com/tutorials/java-reflection/constructors.html)
+*у слайдах присутні приклади
+*/
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+
+public class PersonFactory {
+
+    public Person createPerson() throws ClassNotFoundException, NoSuchMethodException, InvocationTargetException, InstantiationException, IllegalAccessException {
+
+        Class newPerson = Class.forName(Person.class.getName());
+        Class[] par = {String.class, int.class, String.class};
```

Comment:
It's better to give more clear names to variables.
Examples:
constructorParameter
initParamTypes
..